dynomitemanager/src/main/resources/dynomitemanager.properties

```diff
@@ -92,8 +92,14 @@ dm.dynomite.storage.preconnect=true
 # ENV: DM_DYNOMITE_MULTI_DC
 dm.dynomite.multi.dc=true
 
-#florida.dyno.mbuf.size=16384
-#florida.dyno.allocated.messages=200000
+# Dynomite memory buffer (mbuf) size in bytes
+# OLD: florida.dyno.mbuf.size=16384
+# ENV: DM_DYNOMITE_MBUF_SIZE
+dm.dynomite.mbuf.size=16384
```

Comment:
yannis explained me the situation here. LGTM. Hopefully other folks are not using these properties. How would we know if we are not breaking anything for anyone here?